R/corridor.R

```diff
@@ -57,8 +64,12 @@ delineate_corridor <- function(
   edge_2 <- corridor_edge(network_2, end_points, edges_init[2], corridor_init,
                           max_iterations)
 
-  # Cap the corritor and return it
-  cap_corridor(c(edge_1, edge_2), capping_method, network)
+  # Cap the corritor
+  corridor <- cap_corridor(c(edge_1, edge_2), capping_method, network)
+
+  # Loops along the edges (or capping segments) form additional polygons, that
+  # can be discarded by selecting only the largest feature
```

Comment:
It is not clear to me what "loops along the edges (or capping segments) form additional polygons" are

Reply:
I agree with you - when re-reading this now it sounds pretty cryptic 😅 .  What I noticed for a couple of cities is that the corridor edges would cross with one of the "capping" edges forming a sort of a loop-hole on the corridor boundary. When polygonized, this would lead to a small side polygon at the end of the corridor, which we ideally want to remove. Will try to tackle this better here.